style.css

```diff
@@ -0,0 +1,290 @@
+html {
+  font-family: Roboto, system-ui, -apple-system, BlinkMacSystemFont;
+}
+
+body {
+  margin: 0;
+}
+
+ul,
+ol {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-grey: #757575;
+  --color-blue: #2196f3;
+  --color-white: #ffffff;
+  --background-color: #2f303a;
+}
+
+.container {
+  width: 1170px;
+  margin: 0 auto;
+}
+
+.header__content {
+  display: flex;
+}
+
+.header__logo {
+  margin-right: 93px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+
+  font-family: Raleway, Arial, Helvetica, sans-serif;
+  font-size: 26px;
+  font-weight: 700;
+  letter-spacing: 0.03px;
+}
+
+.logo__web,
+.is-active,
+.nav__item:hover,
+.contacts__link:hover {
+  color: var(--color-blue);
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+
+  gap: 50px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+
+  height: 80px;
+
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: 0.02px;
+}
+
+.header__contacts {
+  margin-left: auto;
+  gap: 50px;
+}
+
+.contacts {
+  display: flex;
+}
+
+.header--link {
+  display: flex;
+  align-items: center;
+
+  height: 80px;
+
+  font-weight: 500;
+  font-size: 14px;
+  font-style: normal;
+  letter-spacing: 0.02px;
+
+  color: var(--color-grey);
+}
+
+.banner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  height: 600px;
+
+  background-color: var(--background-color);
+}
+
+.banner__title {
+  margin: 0 0 30px;
+
+  width: 696px;
+
+  font-size: 44px;
+  font-weight: 900;
+  line-height: 60px;
+  letter-spacing: 0.06px;
+
+  text-align: center;
+  text-transform: uppercase;
+
+  color: var(--color-white);
+}
+
+.banner__button {
+  height: 50px;
+  width: 216px;
+
+  color: var(--color-white);
+  background-color: var(--color-blue);
+  border: 1px solid var(--color-blue);
+  border-radius: 4px;
+}
+
+.button__link {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 30px;
+  letter-spacing: 0.06px;
+}
+
+.features__content {
+  display: flex;
+}
+
+.features__list {
+  margin: 0;
+  padding: 94px 0;
+
+  display: flex;
+  gap: 30px;
+}
+
+.feature__title {
+  margin: 0;
+
+  line-height: 16px;
+
+  font-size: 14px;
+  font-weight: 700;
+  letter-spacing: 0.03px;
+}
+
+.feature__subtitles {
+  margin: 10px 0 0;
+
+  line-height: 24px;
+
+  font-size: 14px;
+  letter-spacing: 0.03px;
+
+  color: var(--color-grey);
+}
+
+.title {
+  margin: 0;
+  padding: 0;
+
+  text-align: center;
+
+  line-height: 42px;
+
+  font-size: 36px;
+  font-weight: 700;
+  letter-spacing: 0.03px;
+}
+
+.services__list {
+  margin: 0;
+  padding: 50px 0 94px;
+
+  display: flex;
+  gap: 30px;
+}
+
+.service__image {
+  width: 370px;
+}
+
+.team {
+  margin: 0;
+  padding: 94px 0;
+
+  background-color: #f5f4fa;
+}
+
+.team__list {
+  margin: 50px 0 0;
+  padding: 0;
```

Comment:
you already reset the padding in the **`RESET`**  region, no need to do it twice